src/Tab.tsx

```diff
@@ -2,9 +2,9 @@ import React from 'react'
 
 import { ParamList, TabOptions } from './types'
 
-export interface TabProps<T extends ParamList> {
+export type TabProps<T extends ParamList> = {
```

Comment:
I changed to `type` instead of `interface` because the docs scrip doesn't like interfaces.